### PR TITLE
changes to make tooltip consistent

### PR DESCRIPTION
**Pull request by pravesh-sharma**
Changed tooltip to look like other parts of the application so as to maintain consistency.

### CODE REVIEW

File: Code/frame.js [hunk 1]
```diff
@@ -991,6 +993,20 @@ define([
...
+                        $button.attr('data-toggle','tooltip')
+                        //to display tooltip with shortcut
+                        if(buttonData.key){
+                            $button.attr('data-html','true')
+                            $.each(buttonData.key,function(index,data) {
+                                var child = $('<div class="tooltip-shortcut-key">')
+                                child.text(data)
+                                $shortcutKeyContainer.append(child)
+                            })
+                            $tooltipConatiner.text(buttonData.tip)
```
Comment (adityatoshniwal) [inline Code/frame.js]: `tooltipConatiner` should be `tooltipContainer`

File: Code/frame.js [hunk 2]
```diff
@@ -991,6 +993,20 @@ define([
                         if(buttonData.ariaLabel) {
                             $button.attr('aria-label', buttonData.ariaLabel);
                         }
+                        //atribute to display tooltip
+                        $button.attr('data-toggle','tooltip')
+                        //to display tooltip with shortcut
+                        if(buttonData.key){
+                            $button.attr('data-html','true')
```
Comment (adityatoshniwal) [inline Code/frame.js]: As per our discussion, it was decided to use html for all the tooltips. Irrespective of shortcut keys.

File: Code/frame.js [hunk 3]
```diff
@@ -1027,6 +1043,13 @@ define([
 
                 panel.__update();
 
+                //enabling tooltip after panel creation
+                $('[data-toggle="tooltip"]').tooltip({
```
Comment (adityatoshniwal) [inline Code/frame.js:1047]: Use html for all.

File: Code/frame.js [hunk 4]
```diff
@@ -991,6 +993,20 @@ define([
                         if(buttonData.ariaLabel) {
                             $button.attr('aria-label', buttonData.ariaLabel);
                         }
+                        //atribute to display tooltip
+                        $button.attr('data-toggle','tooltip')
+                        //to display tooltip with shortcut
+                        if(buttonData.key){
+                            $button.attr('data-html','true')
+                            $.each(buttonData.key,function(index,data) {
+                                var child = $('<div class="tooltip-shortcut-key">')
```
Comment (adityatoshniwal) [inline Code/frame.js]: add wc classes to allow overriding the styles.

File: Code/panel.js [hunk 5]
```diff
@@ -307,6 +307,21 @@ define([
             return false;
         },
 
+        /**
+         * updates button in the panel for tooltip.
+         * @function module:wcPanel#updateButton
+         * @param {Array} buttons - List of buttons to update.
+         * @returns {Boolean} - Success or failure.
+         */
+        updateButton: function (buttons) {
```
Comment (adityatoshniwal) [inline Code/panel.js]: Why are you passing the complete list to `updateButton` and replacing the button list ? This is function is not re-usable at all. It should allow changing data for a single button.

File: package.json [hunk 6]
```diff
@@ -9,6 +9,7 @@
   "main": "Build/wcDocker.min.js",
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.14.0",
+    "bootstrap": "^5.2.1",
```
Comment (adityatoshniwal) [inline package.json]: Use "bootstrap": "^4.3.1", as used in pgAdmin.

File: Code/frame.js (hunk 1 above)
Comment (adityatoshniwal) [reply Code/frame.js]: This is not done.

Comment (pravesh-sharma) [reply Code/frame.js]: Screenshot 2022-09-22 at 2 26 24 PM

I have updated it in may latest commit, also can see it in files changed

File: Code/frame.js (hunk 2 above)
Comment (adityatoshniwal) [reply Code/frame.js]: This is not done.

File: Code/frame.js (hunk 1 above)
Comment (adityatoshniwal) [reply Code/frame.js]: OK able to see now.

**Comment by adityatoshniwal** [comment]
The changes looks good to me. @dpage-edb Can you please squash and merge the PR ?

**Pull request merged**